test/trigger/test_trigger.js

```diff
@@ -108,6 +108,64 @@ let testUnnecessaryGroupRegex = function(file, contents) {
   }
 };
 
+let testInvalidCapturingGroupRegex = function(file, contents) {
+  let json = eval(contents);
+
+  for (let i in json[0].triggers) {
+    let currentTrigger = json[0].triggers[i];
+
+    let containsMatches = false;
+    let triggerFunctions = [
+      'alarmText',
+      'alertText',
+      'condition',
+      'delaySeconds',
+      'infoText',
+      'preRun',
+      'run',
+      'suppressSeconds',
```

Comment:
durationSeconds, sound, soundVolume, disabled, groupTTS

Sorry for poor documentation.

It'd be nice to have this list somewhere such that it could also be used to verify that there aren't misspelled trigger entries.